controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -343,12 +346,42 @@ func (r *ReplicationGroupReconciler) processLastActionResult(ctx context.Context
 		}
 	}
 
+	if strings.Contains(group.Status.LastAction.Condition, "FAILOVER_REMOTE") {
+		if err := r.processFailoverEvent(ctx, group, remoteClient, log); err != nil {
+			return err
+		}
+	}
+
+	if strings.Contains(group.Status.LastAction.Condition, "UNPLANNED_FAILOVER_LOCAL") {
```

Comment:
PowerMax driver supports more failover actions. Maybe we can look for FAILOVER action keyword?
https://dell.github.io/csm-docs/docs/deployment/helm/modules/installation/replication/powermax/#basic-site-specific-actions
https://github.com/dell/dell-csi-extensions/blob/main/replication/replication.proto#L151

Reply:
Due to the focus on testing with PowerFlex, I would say that it is best to leave the action processing based on the failovers that are supported. Also, depending on the failover type, different parameters are passed.

In the future, we can enhance this.

Reply:
We have to update this if we are to qualify PowerMax driver for this feature. 
As per my understanding, FAILOVER_LOCAL and UNPLANNED_FAILOVER_REMOTE are same as the 2 actions that are already covered, it is just to run on the other side of RG. I do not have an idea of why 'failover without swap' actions was introduced.

Reply:
If the changes are minor, I would say let's add that when qualifying with PowerMax. By not adding it now, we don't affect the usability or functionality of other drivers failover flow.